client/dapp_chain_rpc_client.go

```diff
@@ -426,6 +426,18 @@ func (c *DAppChainRPCClient) GetEvmTxReceipt(txHash []byte) (ptypes.EvmTxReceipt
 	return receipt, err
 }
 
+func (c *DAppChainRPCClient) GetContractEvents(fromBlock, toBlock uint64) (ptypes.ContractEventsResult, error) {
+	params := map[string]interface{}{
+		"fromBlock": strconv.FormatUint(fromBlock, 10),
+		"toBlock":   strconv.FormatUint(toBlock, 10),
```

Comment:
I think we also need `contract` as another parameter since `contractevents` has it. It can be blank string if we don't want to filter by contract name.

Reply:
We also need  another method to query the block height to really get rid of those JSONRPC client that we're currently using.

`func (c *DAppChainRPCClient) GetBlockHeight() (uint64, error)`

Reply:
Done

Reply:
`contract` parameter does not work though. I think we need to have a fix for it.

Reply:
Yeah. The name might be mapped incorrectly in loomchain.